src/vstart.sh

```diff
@@ -692,12 +693,6 @@ EOF
         ;rgw crypt vault secret engine = transit
         ;rgw crypt vault prefix = /v1/transit/export/encryption-key/
 
-$extra_conf
-EOF
-wconf <<EOF
-[cephfs-shell]
```

Comment:
I was rather hoping this would work. Can you dig into this more?

Reply:
Sure.